### ISSUE TITLE
Debates

### ISSUE THREAD

**Issue by stpaultim** [OpenKratia/openkratia#4]
In https://github.com/OpenKratia/openkratia/issues/2, Debates are listed as one of the two initial components. What will debates look like? What do you have in mind for a minimum viable product?

**Comment by robertgarrigos** [comment]
> Is a "debate" similar to a "topic" in a forum?

Yes, that's exactly what I was thinking of. This is something I don't like from Decidim: that debates don't look like forum topics (mainly because of their grid structure).

Don't know yet whether we will be able to use the forum module or will have to create a new one. However, due to the specific features I believe OpenKratia needs, I think it would be easier to have a new Debates module.

What are these specific features:

1. Navigation: (this comes from Decidim, which I like) every content is related, visually, to its parent space and its sibling components. Not just when you are browsing content, but also when editing or creating it.
2. Sorting: (Decidim lacks, partially, this feature, where you can sort comments, but not debates) Sorting means rating, this is, the avility to rate not just comments, but also debates (and proposals), so you can list them as much rated and, thus, most interesting to the community.

For me, then, a minimum viable Debates module would be one with this two key features.
